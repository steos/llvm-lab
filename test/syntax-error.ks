
	void main() {
		int foo = 3
	}