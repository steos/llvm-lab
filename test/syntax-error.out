Parse error in ./test/syntax-error.ks on line 4 at offset 1: missing token, expected SEMICOLON
